tests/test-dirs/escape_region/illegal-usage.t

```diff
@@ -0,0 +1,14 @@
+  $ cat >local.ml <<EOF
```

Comment:
Just to reduce some clutter, could the two new tests be combined into one .t file? And I would put these in the `locate` directory -- no need to create a new directory just for this case.